python/docs/booking-demo.md

```diff
@@ -32,7 +32,7 @@ restaurant = Agent(
 
 fund_agent_if_low(restaurant.wallet.address())
 ```
-The protocols `query_proto` and `book_proto` are built from message handlers in the same way as agents. See [query protocol](https://github.com/fetchai/uAgents/blob/main/python/examples/09-booking-protocol-demo/protocols/query.py) and [book protocol](https://github.com/fetchai/uAgents/blob/main/python/examples/09-booking-protocol-demo/protocols/book.py) for the details and logic behind these protocols, but for now we will simply import them. You will need to add these files inside a `protocols` folder in the same directory you are running your agent. See [agent protocols](agent-protocols.md) for more information.
+The protocols `query_proto` and `book_proto` are built from message handlers in the same way as agents. You will need to add these files inside a `protocols` folder in the same directory you are running your agent. See [agent protocols](agent-protocols.md) for more information.
```

Comment:
The `09-booking-protocol-demo` wasn't in the [planned](https://github.com/fetchai/uAgents/pull/309) overhaul of the examples and so was not moved into the [Examples](https://github.com/fetchai/uAgent-Examples/tree/main/1-uagents/uagent-101) repo.

I suggest we refactor this doc to either refer to another example or add this example back in the new repo and replace the links.

Same below.

Reply:
@5A11 since this doc is going through the booking example that we don't plan to include anymore I think we can just delete this file. Also we already explain protocols in agent-protocols.md

Reply:
Sounds good to me 👍🏻